app/containers/ITunesContainer/index.js

```diff
@@ -48,6 +94,12 @@ export function ITunesContainer({ dispatchITunesSongs, dispatchClearITunesSongs,
           onSearch={(searchText) => debouncedHandleOnChange(searchText)}
         />
       </CustomCard>
+      <Skeleton loading={loading} active></Skeleton>
+      <Container data-testid="container">
+        {iTunesData?.results?.map((songDetails, index) => (
```

Comment:
use the For component

Reply:
Yes. Will update the code .Will also make the changes needed to  handle Error conditions